txpool/blocklist_test.go

```diff
@@ -0,0 +1,169 @@
+package txpool
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vechain/thor/v2/thor"
+)
+
+func SetupFile(t *testing.T, testFilePath string, dummyData string) {
```

Comment:
How about storing the test file in the OS's [temporary](https://pkg.go.dev/os#CreateTemp) dir?